src/pages/workspace/WorkspaceSettingsPage.js

```diff
@@ -55,20 +56,31 @@ class WorkspaceSettingsPage extends React.Component {
         if (this.props.policy.isPolicyUpdating) {
             return;
         }
-        const name = values.name.trim();
         const outputCurrency = values.currency;
-        Policy.updateGeneralSettings(this.props.policy.id, name, outputCurrency);
+        Policy.updateGeneralSettings(this.props.policy.id, values.name, outputCurrency);
         Keyboard.dismiss();
     }
 
     validate(values) {
         const errors = {};
-        if (!values.name || !values.name.trim().length) {
```

Comment:
Will it cause any issue if we don't check `!values.name`? If for any reason `values.name` is not string, then `.trim()` will throw an error.

Reply:
I think that check is unnecessary as `values.name` is always guaranteed to be string from the `Form` component.

Reply:
Thanks for confirming @youssef-lr. Was just highligting.